lib/backend/k8s/k8s_fv_test.go

```diff
@@ -943,6 +943,7 @@ var _ = Describe("Test Syncer API for Kubernetes backend", func() {
 		})
 
 		By("applying a new object", func() {
+			gc.Revision = nil // Revision should not be specified when creating.
```

Comment:
It's only test code, but I prefer to keep comments above the things they're referring to than inline.